_posts/2019-06-03-GSoC - FirstSteps.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Google Summer of Code 2019: First Steps"
+id: gsoc-2019-first-steps
+author: "[JabRef Developers](https://github.com/JabRef/jabref/blob/master/DEVELOPERS)"
+bg: jabref-font
+color: white
+---
+
+Hello!
+My name is David Méndez, and this summer I will be working on the [LaTeX Integration project for the Google Summer of Code](https://summerofcode.withgoogle.com/projects/#6055042405105664).
+The goal of this project is to facilitate working with LaTeX for the end user.
+I will develop several tools that analyze and check LaTeX text documents: a search for bibliographic entries (which are used, how many times and where); a counting tool for words, characters and quotes per document; a validator of TEX files to solve issues; and a tool to import entries from TEX files.
```

Comment:
Make this a bullet list and maybe include one or two pictures from your prototypes.